test/language/module-code/namespace/internals/get-own-property-sym.js

```diff
@@ -28,7 +28,7 @@ desc = Object.getOwnPropertyDescriptor(ns, Symbol.toStringTag);
 assert.sameValue(desc.value, ns[Symbol.toStringTag]);
 assert.sameValue(desc.enumerable, false, 'Symbol.toStringTag enumerable');
 assert.sameValue(desc.writable, false, 'Symbol.toStringTag writable');
-assert.sameValue(desc.configurable, true, 'Symbol.toStringTag configurable');
+assert.sameValue(desc.configurable, false, 'Symbol.toStringTag configurable');
```

Comment:
In the text? The others say "enumerable" and "writable" despite checking for non-enumerable and non-writable, was my thinking. Could change all of them.